dev/com.ibm.ws.session.cache/src/com/ibm/ws/session/store/cache/CacheStoreService.java

```diff
@@ -149,7 +164,18 @@ private void processConfiguration(Map<String, Object> props) {
     }
 
     protected void modified(Map<String, Object> config) {
-        processConfiguration(config);
+        if (checkpointPhase != CheckpointPhase.INACTIVE && checkpointPhase.restored()) {
+            for (Map.Entry<String, Object> entry: config.entrySet()) {
+                String key = entry.getKey();
+                Object newValue = entry.getValue();
+                
+                Object oldValue = oldConfigurationProperties.get(key);
```

Comment:
You should not need to have a copy of the `oldConfigurationProperties` here for this check. At this point `configurationProperties` and `oldConfigurationProperties` should be identical.  You should be able to just check against `configurationProperties` values here.

Reply:
Done, thanks!